commands/fly.go

```diff
@@ -3,7 +3,8 @@ package commands
 import "github.com/concourse/fly/rc"
 
 type FlyCommand struct {
-	Target rc.TargetName `short:"t" long:"target" description:"Concourse target name"`
+	Target  rc.TargetName  `short:"t" long:"target" description:"Concourse target name"`
+	Targets TargetsCommand `command:"targets" description:"View all targets save to your .flyrc"`
```

Comment:
This should have a `ts` alias. I also don't like mentioning `.flyrc` here - maybe just "List the saved targets"?